app/packages/core/src/plugins/SchemaIO/components/FileView.tsx

```diff
@@ -86,3 +87,7 @@ function fileToBase64(
     fileReader.onerror = (error) => resolve({ error });
   });
 }
+
+function stripBase64Prefix(data: string): string {
+  return data.replace(/^data:image\/[a-z]+;base64,/, "");
```

Comment:
This could be other MIME types too, like `data:application/json;base64,XXX`.

In general I think splitting on the first occurrence of `base64,` is sufficient